apps/aecontract/src/aect_ring.erl

```diff
@@ -53,6 +56,15 @@ hexstring_encode(Code) ->
            || <<X:4, Y:4>> <= Code >>,
     <<"0x", CodeAsHexString/binary >>.
     
+hexstring_decode(Code) ->
+    <<"0x", CodeAsHexString/binary >> = Code,
+    0 = bit_size(CodeAsHexString) rem 8,
```

Comment:
Would it get past the `/binary` at the previous line if it wasn't full bytes?

Reply:
```
1> Bin = <<"0x">>.
<<"0x">>
2> <<"0x", Rest/binary>> = Bin.
<<"0x">>
3> Rest.
<<>>
```

Reply:
for completeness
```
4> bit_size(Rest).
0
```